WordPress/Tests/WP/AlternativeFunctionsUnitTest.inc

```diff
@@ -171,3 +171,14 @@ file_get_contents(\ABSPATH . 'wp-admin/css/some-file.css');
 file_get_contents(MyNamespace\ABSPATH . 'wp-admin/css/some-file.css');
 file_get_contents(\MyNamespace\ABSPATH . 'wp-admin/css/some-file.css');
 file_get_contents(namespace\ABSPATH . 'wp-admin/css/some-file.css');
+
+/*
+ * Safeguard that the sniff does not incorrectly ignore class methods/constants with the same
+ * name as WordPress global functions/constants when used in file_get_contents().
+ */
```

Comment:
These tests cover the change, but might be good to vary the constants/functions being referenced in the tests ? And for the functions to include some case variation ?

Reply:
Yeah, but depending on the sniff, some functions may have special handling, while others do not. If all "namespaced name" tests only relate to the one function, it can easily be missed that special handling for another function is broken in the context of namespaced names (typically FQN global function).

I'm not trying to make your life more difficult, I'm trying to make it easier by having tests which will actually show what needs fixing for PHPCS 4.0.

Reply:
That is fair. I went ahead and added variation for the tests that safeguard that the sniff does not incorrectly ignore class methods/constants with the same name as WordPress global functions/constants when used in file_get_contents(). I did not do the same for the tests added in the other commit, as I imagine they will be removed from this PR pending what we decide in the other comment thread in this PR. I do plan to add variation to those tests as well.